r/tests/testthat/helper-skip.R

```diff
@@ -70,6 +70,10 @@ skip_if_not_dev_mode <- function() {
 }
 
 skip_if_not_running_large_memory_tests <- function() {
+  if (force_tests()) {
```

Comment:
Did you add this back in accidentally? Or am I missing something?

Reply:
I added it back on purpose (to test without the bug in `force_tests()`). Given the bug in `force_tests()`, `force_tests()` can never return `true` (AFAIU). So, technically, that line is inconsequential.

Reply:
The downside of ☝🏻 this is that, if:

```r
  if (force_tests()) {
    return()
  }
```
was never ran (since `force_test()` would never be `TRUE`), then the issue won't be solved by removing those lines from `skip_if_not_running_large_memory_tests()`.

Reply:
For testing that `force_tests()` does actually force all the tests, we might put in an explicitly bad test (I usually do `expect_true(false)` or the like). IIUC, the OOM here isn't 100% deterministic so relying on that to be the thing that fails might be a bit frustrating

Reply:
I tested `force_tests()` locally and it now works. Given the fact that we can't reproduce the OOM failure reliably, I think the best path for this PR is:
* add an `expect_true(false)` test in a commit and show that it causes a CI failure, and then
* undo the ☝🏻 commit.